<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>igdGuiMainUi</class>
 <widget class="QMainWindow" name="igdGuiMainUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UPnP IGD GUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="buttonScan">
        <property name="text">
         <string>&amp;Scan</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelStatus">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="lANIPAddressLabel">
        <property name="text">
         <string>LAN IP address</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lANIPAddressLineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="externalIPAddressLabel">
        <property name="text">
         <string>External IP address</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="externalIPAddressLineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListView" name="forwardList"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="buttonNew">
        <property name="text">
         <string>&amp;New</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonDetails">
        <property name="text">
         <string>&amp;Details</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonDelete">
        <property name="text">
         <string>D&amp;elete</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonRefresh">
        <property name="text">
         <string>&amp;Refresh</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonQuit">
        <property name="text">
         <string>&amp;Quit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
